tools/Microsoft.AspNetCore.Mvc.Razor.ViewCompilation-x86/Microsoft.AspNetCore.Mvc.Razor.ViewCompilation-x86.csproj

```diff
@@ -0,0 +1,20 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Was a bit curious about this one too. Waiting on @pranavkm's feedback on my comment [here](https://github.com/aspnet/MvcPrecompilation/pull/130#discussion_r118806771).